### PR TITLE
translation(zyc): add gym_super_mario_bros English file 

### PR DESCRIPTION
**Pull request by sylviai**

### CODE REVIEW

**Comment by TuTuHuss** [comment]
add file name into ./index.rst

**Comment by TuTuHuss** [comment]
rename pull request name like other opening pull requests

File: gym_super_mario_bros.rst [hunk 1]
```diff
@@ -0,0 +1,389 @@
+Gym-Super-Mario-Bros
```
Comment (TuTuHuss) [inline gym_super_mario_bros.rst:1]: delete this file

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 2]
```diff
@@ -0,0 +1,389 @@
+Gym-Super-Mario-Bros
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Overview
+=======
+Here is the "Super Mario Bros" series of games, in which players need to control Mario to move and jump, avoid the pits and enemies in the process of leading to the end, gain more gold coins to get higher scores. This game also has many interesting props to enhance player experiences.`gym-super-mario-bros <https://github.com/Kautenja/gym-super-mario-bros>`_ , this environment is encapsulated from "Super Mario Bros" of Nintendo after OpenAI Gym.
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:6]: format problem

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+
+    # Start 1-4 level
+    gym_super_mario_bros -e 'SuperMarioBrosRandomStages-v0' -m 'human' --stages '1-4'
+
+
+Action Space
+----------
+
+The action space of gym-super-mario-bros contains the whole 256 discrete actions from Nintendo.
+To compress this size (and to facilitate learning by the intelligences), the environment provides the action wrapper `` JoypadSpace `` by default to reduce the action dimension: the optional set of actions and their meanings are as follows:
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:105]: format problem

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+
+.. image:: ./images/mario_v3.png
+   :align: center
+   :scale: 70%
+
+Reward Space
+-----------
+We hope Mario could more likely to move to the **right side** , and move **faster** to the end successfully, the setting of the reward for each frame consists of three parts as follows:
+
+1. `` v ``:represents the difference in Mario's x-coordinate (which can be interpreted as the velocity to the right) between two consecutive frames, with positive and negative.
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:198]: format problem

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+
+
+2. `` c ``:the time used per frame, simply understood as a negative REVERSE for each frame, is used to push the intelligence to reach the end faster.
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:201]: format problem

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+
+
+3. `` d ``:penalty for death, giving a high penalty of -15 if Mario dies.
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:204]: format problem

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 7]
```diff
@@ -0,0 +1,389 @@
...
+
+
+Total reward `` r = v + c + d ``
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:207]: format problem

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 8]
```diff
@@ -0,0 +1,389 @@
...
+
+Reward being clipped to `` (-15,15) ``
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:209]: format problem

File: source/13_envs/Gym-Super-Mario-Bros.rst [hunk 9]
```diff
@@ -0,0 +1,389 @@
...
+
+- Mario wins
+  
+- Mario is dead
+  
+- Countdown ends
+
+Additional information contained in info
+----------------------------
+At each step of interaction with the environment , the environment returns the ` info ` dictionary, which contains information about the coins acquired, the current accumulated score, the time remaining, and Mario's current coordinates. The details are as follows:
```
Comment (TuTuHuss) [inline source/13_envs/Gym-Super-Mario-Bros.rst:224]: format problem

**Comment by TuTuHuss** [comment]
use the same file name format and case with chinese version

File: source/13_envs/gym-super-mario-bros.rst [hunk 10]
```diff
@@ -0,0 +1,389 @@
...
+    File "test_mario.py", line 13, in <module>
+        state, reward, done, info = env.step(env.action_space.sample())
+    File "/Users/wangzilin/opt/anaconda3/envs/mario_test/lib/python3.8/site-packages/nes_py/wrappers/joypad_space.py", line 74, in step
+        return self.env.step(self._action_map[action])
+    File "/Users/wangzilin/opt/anaconda3/envs/mario_test/lib/python3.8/site-packages/gym/wrappers/time_limit.py", line 50, in step
+        observation, reward, terminated, truncated, info = self.env.step(action)
+    ValueError: not enough values to unpack (expected 5, got 4)
+
+Due to the updates of gym-super-mario-bros code base cannot keep up with the updates of gym code base sometimes, while executing `pip install gym-super-mario-bros`, the latest gym would be installed by default.The solution is to downgrade gym.
+Here gym-super-mario-bros version is 7.4.0, gym version is 0.26.2.We may choose to downgrade gym version to 0.25.1 to solve problems.
```
Comment (PaParaZz1) [inline source/13_envs/gym-super-mario-bros.rst:63]: a space after `gym version is 0.26.2.`

File: source/13_envs/gym-super-mario-bros.rst [hunk 11]
```diff
@@ -0,0 +1,389 @@
...
+
+2. ``c``:the time used per frame, simply understood as a negative REVERSE for each frame, is used to push the intelligence to reach the end faster.
+
+
+3. ``d``:penalty for death, giving a high penalty of -15 if Mario dies.
+
+
+Total reward ``r = v + c + d``
+
+Reward being clipped to ``(-15,15)``
```
Comment (PaParaZz1) [inline source/13_envs/gym-super-mario-bros.rst:209]: grammar error?

File: source/13_envs/gym-super-mario-bros.rst [hunk 12]
```diff
@@ -0,0 +1,389 @@
...
+
+    # run 1 episode
+    env.reset()
+    while True:
+        state, reward, done, info = env.step(env.action_space.sample())
+        if done or info['time'] < 250:
+            break
+    print("Your mario video is saved in {}".format(video_dir_path))
+    try:
+        # 环境的析构函数有问题，故需要异常来避免报错
```
Comment (PaParaZz1) [inline source/13_envs/gym-super-mario-bros.rst:299]: translate Chinese to English

File: source/13_envs/gym-super-mario-bros.rst [hunk 13]
```diff
@@ -0,0 +1,389 @@
+Gym-Super-Mario-Bros
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Overview
+=======
```
Comment (PaParaZz1) [inline source/13_envs/gym-super-mario-bros.rst:5]: the underline notation should be longer than the above text

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**